src/data/tours/tour_0/tour.json

```diff
@@ -4,143 +4,130 @@
   "tourDuration": "102",
   "locations": [{
       "name": "vive la tarte",
-      "summary": "Lofty, industrial-chic Belgian bakery with an open kitchen for tarts, plus coffee, tea & juice.",
-      "spiritualSignificance": "Deliciousness",
+      "short_address": "1160 Howard St",
       "googleMapsParameter": {
         "name": "Vive la tarte",
         "address": "1160 Howard St, San Francisco, CA 94103"
       },
-      "locationImage": "http://frenchbydesign.frenchbydesign.netdna-cdn.com/wp-content/uploads/2015/12/vive-la-tarte_frenchbydesign_4.jpg",
-      "tasks": [
+      "locationImage": "images/vive-la-tarte.jpg",
+      "thumbnail": "images/vive-la-tarte.jpg",
+      "details": [
+        {
+          "title": "Overview",
+          "text": "Lofty, industrial-chic Belgian bakery with an open kitchen for tarts, plus coffee, tea & juice.",
+          "carousel": [
+              "images/vive-la-tarte.jpg",
+              "images/vive-la-tarte.jpg",
+              "images/vive-la-tarte.jpg"
+          ]
+        },
         {
-          "icon": " ",
-          "text": "Eat a tart here."
+          "title": "Spiritual Needs",
+          "text": "Eat a tart here too. Deliciousness."
         }
       ]
     },
     {
       "name": "Chez Maman",
-      "summary": "Cozy, informal eatery serving French bistro classics ranging from burgers & crêpes to mussels.",
-      "spiritualSignificance": "We should pray for all the French people who come here.",
+      "short_address": "401 Gough St",
       "googleMapsParameter": {
         "name": "Chez Maman",
         "address": "401 Gough St, San Francisco, CA 94102"
       },
-      "locationImage": "http://www.thejudge.ch/wp-content/uploads/2014/06/thejudge-LIP-HIMALAYA-1.jpg",
-      "tasks": [
-        {
-          "icon": " ",
-          "text": "Order some mussels."
-        },
+      "locationImage": "images/chez-maman.jpg",
```

Comment:
Should `locationImage` go inside the "details" block?

Reply:
Decided that we'll keep all objects in the JSON the same type and locationImage doesn't fit that type.